exercises/complex-numbers/canonical-data.json

```diff
@@ -323,6 +323,12 @@
             "z": [1, 0]
           },
           "expected": ["e", 0]
+        },
+        {
+          "description": "Exponential of a number with real and imaginary part",
+          "property": "exp",
+          "input": ["ln(2)", "pi"],
```

Comment:
This seems to be missing the `{ z: }` bits.

Reply:
These changes were originally made before the adaptation to the new schema; I will correct it.